src/lib/konnectors.js

```diff
@@ -152,6 +152,7 @@ function waitForKonnectorReady (cozy, konnector, timeout) {
           }
         })
         .catch(error => {
+          if (error.status === 404) return // keep waiting
```

Comment:
👍 

How did we have the whole thing working until now ?

Reply:
I don't know but maybe because we don't use a lot of new instances to test in dev and integration (personally I didn't reset my dev instance frequently) and in production konnector installations are better handled. But I think there was a bug about that: sometimes we didn't understand why we have to connect twice MAIF to get it worked.